### ISSUE TITLE
Error installing requirements.txt "etils >= 0.3.2" line

### ISSUE THREAD

**Issue by letyrodridc** [google-research/meta-dataset#87]
Doing 
```
pip3 install -r requirements.txt 
```
I'm getting:
```
ERROR: Could not find a version that satisfies the requirement etils>=0.3.2 (from versions: none)
ERROR: No matching distribution found for etils>=0.3.2
```
Not sure what is the etils library and it maybe fixed someway in the requirements.txt.

**Comment by vdumoulin** [comment]
Thank you for flagging this! See #82. I will send out a fix shortly.

**Comment by vdumoulin** [comment]
Fixed via 8fbfc63.
